srcpkgs/cegui/template

```diff
@@ -14,7 +14,6 @@ makedepends="libglvnd-devel SDL2-devel SDL2_image-devel libxml2-devel boost-deve
  devil-devel glfw-devel glm ois-devel silly-devel fribidi-devel lua53-devel minizip-devel
  $(vopt_if irrlicht irrlicht-devel)
  $(vopt_if ogre libogre-devel)
- $(vopt_if python python-devel)
  $(vopt_if gtk gtk+-devel)"
```

Comment:
what would dropping gtk2 mean?

Reply:
http://static.cegui.org.uk/docs/0.8.7/dependencies.html:

>  GTK2 (http://www.gtk.org/)
>  Required to enable the GTK2 based renderer selector used in the samples.

So, probably fine

Reply:
ok, let's drop it then